ksim/task/rl.py

```diff
@@ -579,6 +579,10 @@ def log_reward_stats(self, transitions: Transition, reward_generators: Collectio
         for key, value in reward_stats.items():
             self.logger.log_scalar(key=key, value=value, namespace="reward")
 
+        # Logs the total reward.
+        total_reward = jnp.sum(transitions.reward) / num_episodes
```

Comment:
Common normalization is to scale with time and clip:
```suggestion
        jnp.clip(rewards * self.config.dt, 0, 1000)
```

Reply:
would rather not do this